pkg/webhook/v1beta1/experiment/validator/validator_test.go

```diff
@@ -68,6 +68,25 @@ func TestValidateExperiment(t *testing.T) {
 		oldInstance     *experimentsv1beta1.Experiment
 		testDescription string
 	}{
+		// Name
+		{
+			Instance: func() *experimentsv1beta1.Experiment {
+				i := newFakeInstance()
+				i.Name = "1234-test"
+				return i
+			}(),
+			Err:             true,
+			testDescription: "Name is invalid",
+		},
+		{
+			Instance: func() *experimentsv1beta1.Experiment {
+				i := newFakeInstance()
+				i.Name = "test-1234"
+				return i
+			}(),
+			Err:             false,
+			testDescription: "Name is valid",
+		},
```

Comment:
You can omit this test-case, since we pass the valid Experiment here:
https://github.com/kubeflow/katib/blob/2a28caa9db92223afcfde95ed47869d547eef660/pkg/webhook/v1beta1/experiment/validator/validator_test.go#L137

Reply:
Thanks!